src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -61,101 +56,88 @@
                 new Translation2d(-WHEEL_BASE / 2.0, -TRACK_WIDTH / 2.0)
         );
 
-        /* Swerve Current Limiting */
-        public static final int ANGLE_CONTINUOUS_CURRENT_LIMIT = 25;
-        public static final int ANGLE_PEAK_CURRENT_LIMIT = 40;
-        public static final double ANGLE_PEAK_CURRENT_DURATION = 0.1;
-        public static final boolean ANGLE_ENABLE_CURRENT_LIMIT = true;
+        public static final double ANGLE_OPEN_LOOP_RAMP = 2;
+        public static final double ANGLE_CLOSED_LOOP_RAMP = .5;
+
+        public static final double DRIVE_OPEN_LOOP_RAMP = 0.1;
+        public static final double DRIVE_CLOSED_LOOP_RAMP = 0.1;
 
-        public static final SupplyCurrentLimitConfiguration ANGLE_CURRENT_LIMIT = new SupplyCurrentLimitConfiguration(
-                ANGLE_ENABLE_CURRENT_LIMIT, ANGLE_CONTINUOUS_CURRENT_LIMIT, ANGLE_PEAK_CURRENT_LIMIT,
-                ANGLE_PEAK_CURRENT_DURATION
+        private static final SupplyCurrentLimitConfiguration ANGLE_CURRENT_LIMIT = new SupplyCurrentLimitConfiguration(
+                true, 25, 40,
+                0.1
         );
 
-        public static final int DRIVE_CONTINUOUS_CURRENT_LIMIT = 35;
-        public static final int DRIVE_PEAK_CURRENT_LIMIT = 60;
-        public static final double DRIVE_PEAK_CURRENT_DURATION = 0.1;
-        public static final boolean DRIVE_ENABLE_CURRENT_LIMIT = true;
+        //TODO: Add to LocalConstants
+        private static final NeutralMode ANGLE_NEUTRAL_MODE =
+                LOCAL_CONSTANTS.localSwerveConstants.angleNeutralMode.equals(
+                        "Brake") ? NeutralMode.Brake : NeutralMode.Coast;
 
-        public static final SupplyCurrentLimitConfiguration DRIVE_CURRENT_LIMIT = new SupplyCurrentLimitConfiguration(
-                DRIVE_ENABLE_CURRENT_LIMIT, DRIVE_CONTINUOUS_CURRENT_LIMIT, DRIVE_PEAK_CURRENT_LIMIT,
-                DRIVE_PEAK_CURRENT_DURATION
+        public static final MotorConfig ANGLE_MOTOR_CONFIG = new MotorConfig(
+                2,
+                0.5,
```

Comment:
use the parameters you defined for this

Reply:
I deleted them.
the whole point of motor config is to shorten things down
you don't save a int of kP, kI, kD and so on so why save these?

Reply:
So they would be easier to change. Most of the constants here are later put in constructors, but we separate them for better readability.